src/rgw/rgw_sal.h

```diff
@@ -471,15 +471,17 @@ class RGWRadosBucket : public RGWBucket {
 
 class RGWRadosStore : public RGWStore {
   private:
+    struct req_state *s;
```

Comment:
the `RGWRadosStore` is effectively global to radosgw, but there's a `req_state` for every request; so it doesn't make sense to store a single `req_state` pointer here